### PR TITLE
Mazda: send dash alerts

### PR DESCRIPTION
**Pull request by adeebshihadeh**
- [ ] <s>LDW</s> TODO
- [x] steer required
- [x] disable initial hold wheel alert

### CODE REVIEW

File: selfdrive/car/mazda/carstate.py [hunk 1]
```diff
@@ -169,7 +169,6 @@ def get_cam_can_parser(CP):
         ("CTR",              "CAM_LKAS", 0),
         ("ERR_BIT_1",        "CAM_LKAS", 0),
         ("LINE_NOT_VISIBLE", "CAM_LKAS", 0),
-        ("LDW",              "CAM_LKAS", 0),
```
[the remark this answers is not in the document]
Comment (adeebshihadeh) [reply selfdrive/car/mazda/carstate.py:169]: we don't want to forward this

File: selfdrive/car/mazda/carstate.py [hunk 2]
```diff
@@ -104,10 +105,21 @@ def get_can_parser(CP):
...
+      ("BIT3", "CAM_LANEINFO", 0),
+      ("NO_ERR_BIT", "CAM_LANEINFO", 1),
+      ("LANE_LINES", "CAM_LANEINFO", 0),
+      ("S1", "CAM_LANEINFO", 0),
+      ("S1_HBEAM", "CAM_LANEINFO", 0),
     ]
 
     checks = [
       # sig_address, frequency
+      ("CAM_LANEINFO", 2),
```
[the remark this answers is not in the document]
Comment (adeebshihadeh) [reply selfdrive/car/mazda/carstate.py:119]: are you sure? it's 2Hz on our CX9

File: selfdrive/car/mazda/mazdacan.py [hunk 3]
```diff
@@ -59,6 +63,21 @@ def create_steering_control(packer, car_fingerprint, frame, apply_steer, ldw, lk
...
 
+def create_alert_command(packer, cam_msg, ldw, steer_required):
+  values = copy.copy(cam_msg)
+  values.update({
+    # TODO: what's the difference between all these? do we need to send all?
+    "HANDS_WARN_3_BITS": steer_required,
+    "HANDS_ON_STEER_WARN": steer_required,
+    "HANDS_ON_STEER_WARN_2": steer_required,
+
+    # TODO: need to do something about L/R
```
[the remark this answers is not in the document]
Comment (adeebshihadeh) [reply selfdrive/car/mazda/mazdacan.py:74]: yes, but not in the CarControl packet

File: selfdrive/car/mazda/carstate.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Jafaral) [reply selfdrive/car/mazda/carstate.py]: you are right, I thought I was looking at 100ms intervals, it is 500ms

**Pull request merged**